### ISSUE TITLE
Bitbucket triggers broken?

### ISSUE THREAD

**Issue by mgladdish** [Shippable/support#2118]
Yet again over the past few days Shippable has been missing pushes to bitbucket. Some projects will trigger, others will sit for days without noticing that anything's changed (all the while there are several changes per day to the repo).

Still there's no clue as to whether the integration hooks are working or whether there's a problem, regardless of at whose end the problem lies. It's guesswork as to whether I have to come in to Shippable's clunky UI to manually kick off a build or not.

**Comment by kepkin** [comment]
Same here. Our project 5645fd5d1895ca4474237436 misses last build.

**Comment by manishas** [comment]
@a-murphy can we take a look at this asap?

**Comment by kanthchandra** [comment]
Hi @mgladdish , Can you please send the projectId which is not picking up webhooks? We are not able to reproduce this behaviour on our end. 

@mgladdish @kepkin It would also help if you could take a screenshot of the webhooks requests for that project and post it here. You can find the page by clicking on the settings option on bitbucket ui and then clicking on webhooks and then selecting the requests option.

I have put up a sample screenshot, which shows the status code for that particular request. This will help us debug if the problem is on our end. 
![selection_051](https://cloud.githubusercontent.com/assets/1583444/11738051/2852f198-a005-11e5-8fac-eeef0114ebae.png)

**Comment by kepkin** [comment]
@kanthchandra It looks like bitbucket missed one push. I couldn't find a corresponding push for missed build.

**Comment by kanthchandra** [comment]
@kepkin We tried to reproduce this on our end and we kept an eye on the commits and the Shippable Webhook logs on Bitbucket UI and also the builds triggered on Shippable. We found that for some commits (like 1 in 10 or so) bitbucket dint push a webhook to us and there was no corresponding entry for that commit in the Shippable webhook request logs on their UI. 

So we suspect that the problem might be on Bitbucket's end. But we will keep an eye on this.
